src/tools/scale_tool.h

```diff
@@ -65,6 +68,7 @@ Q_OBJECT
 	MapCoordF scaling_center;
 	double reference_length = 0;
 	double scaling_factor   = 1;
+	bool group_scaling = true;
```

Comment:
IMHO `group_scaling` isn't really naming the difference. We are dealing with a group of objects in both cases. How about `use_common_center`?

Reply:
Right. `group_scaling` says too little about the difference between the two modes. `use_common_center` sounds a lot better. If you don't mind, I'd take your proposal a step further. Opting for present participle form of "use" and replacing "common" with "selection" would IMHO enhance fluency in reading the code. For example, the following code lines
```
if (using_selection_center)
    do_something(selection);
```
would read "when using selection center, do something with selection". How does the adjusted name sound to you?

Reply:
👍

Reply:
Pushed updated patch set. Thanks for the smooth cooperation!